packages/validator/templates/1.0.0/endpoints.json

```diff
@@ -0,0 +1,71 @@
+{
+  "__arrayItem": {
+    "name": {},
+    "operation": {
+      "path": {
+        "__regexp": "^\\/[^\\s'\"\\\\]+$"
+      },
+      "method": {
+        "__regexp": "^(get|post)$",
+        "__catch": {
+          "__message": "__fullPath can be either \"get\" or \"post\""
+        }
+      }
+    },
+    "fixedOperationParameters": {
+      "__arrayItem": {
+        "operationParameter": {
+          "name": {},
+          "in": {
+            "__regexp": "^(query|header|path|cookie)$",
+            "__catch": {
+              "__message": "Allowed values in __fullPath are: 'path', 'query', 'header' or 'cookie'"
+            }
+          }
+        },
+        "value": {}
+      }
+    },
+    "reservedParameters": {
+      "__arrayItem": {
+        "name": {
+          "__regexp": "^(_type|_path|_times)$",
+          "__catch": {
+            "__message": "__fullPath: Reserved parameter can be only \"_type\", \"_path\" or \"_times\""
+          }
+        },
+        "__optional": {
+          "fixed": {},
+          "default": {}
+        }
+      }
+    },
+    "parameters": {
+      "__arrayItem": {
+        "name": {
+          "__regexp": "^[^\\s'\"_][^\\s'\"]+$"
+        },
+        "operationParameter": {
+          "name": {},
+          "in": {
+            "__regexp": "^(path|query|header|cookie)$",
+            "__catch": {
+              "__message": "Allowed values in __fullPath are: 'path', 'query', 'header' or 'cookie'"
+            }
+          }
+        },
+        "__optional": {
+          "default": {},
+          "description": {},
+          "required": {},
+          "example": {}
```

Comment:
It seems like OAS/Swagger allows for several other field names:
`deprecated` and `allowEmptyValue`

Reply:
I was following a bit different version of [this document](https://docs.api3.org/pre-alpha/airnode/specifications/ois.html), which doesn't seem to be available anymore, but those fields don't seem to be there as well. I guess we should wait for a more stable version of specifications and then fix these minor inconsistencies.